lib/flat_search_web/live/page_live.ex

```diff
@@ -18,6 +18,8 @@ defmodule FlatSearchWeb.PageLive do
       |> Filters.change_filter(params)
       |> Map.put(:action, :insert)
 
-    {:noreply, assign(socket, changeset: changeset)}
+    flats = Flats.get_flats_by(params)
+
+    {:noreply, assign(socket, changeset: changeset, flats: flats)}
```

Comment:
how about putting `Flats.get_flats_by(params)` just inside the `assign` keymaps below, instead of assigning this to a variable for one-time use?